cds/modules/records/serializers/schemas/project.py

```diff
@@ -75,6 +74,7 @@ class ProjectSchema(StrictKeysSchema):
     title_translations = fields.Nested(TitleTranslationSchema, many=True)
     videos = fields.Nested(ProjectVideoSchema, many=True)
 
-    category = fields.Str()
-    type = fields.Str()
+    date = fields.Str()
+    category = fields.Str(required=True)
+    type = fields.Str(required=True)
```

Comment:
Sorry, that's the one I meant http://marshmallow.readthedocs.io/en/latest/quickstart.html#specifying-attribute-names